invenio_users_resources/services/groups/config.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 TU Wien.
+#
+# Invenio-Users-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""User groups service configuration."""
+
+from invenio_records_resources.services import RecordServiceConfig, \
+    SearchOptions, pagination_links
+
+from ..common import Link
+from ..permissions import UsersPermissionPolicy
+from .results import UserGroupItem, UserGroupList
+
+
+class UserGroupSearchOptions(SearchOptions):
+    """Search options."""
+
+    # TODO search params
+    params_interpreters_cls = SearchOptions.params_interpreters_cls
+
+
+class UserGroupsServiceConfig(RecordServiceConfig):
+    """Requests service configuration."""
+
+    # common configuration
+    permission_policy_cls = UsersPermissionPolicy
+    result_item_cls = UserGroupItem
+    result_list_cls = UserGroupList
+    search = UserGroupSearchOptions
+
+    # specific configuration (TODO is this really not needed?)
```

Comment:
These overrides are needed, so we are good. (otherwise default Record ones are used). In future PR, this is where the faked Record proxy can be placed I think.